api/tests/integration/infrastructure/repositories/participations-for-campaign-management-repository_test.js

```diff
@@ -91,16 +95,16 @@ describe('Integration | Repository | Participations-For-Campaign-Management', fu
 
         // then
         expect(participationsForCampaignManagement[0]).to.be.instanceOf(ParticipationForCampaignManagement);
-        expect(participationsForCampaignManagement[0]).to.deep.equal({
+        expect(participationsForCampaignManagement[0]).to.deep.includes({
           id: campaignParticipation.id,
           lastName: 'King',
           firstName: 'Arthur',
+          userId: campaignParticipation.userId,
+          userFullName: 'Ar Thur',
```

Comment:
🤣

Reply:
👑 ⚔️ 🪨

Reply:
Alors je sais pas si vous êtes au courant mais j'ai ajouté y'a quelques temps des petites méthodes pimpées style 
to.deepEqualInstance qui permet de check en 1 appel et le type et le contenu 😄 . J'ai aussi mis to.deepEqualInstanceOmitting pour les cas où justement on veut ignorer certains attributs à la comparaison.
Voilà je me sauve --> api/tests/tooling/chai-custom-helpers

Reply:
je suis pas sûre que pour notre cas ce soit pertinent, car dans les 2 cas ça fait un new, qui a une logique spécifique. si on met pas les bons champs, on vérifie que undefined = undefined, pas ouf du coup.